### ISSUE TITLE
feature_request(option): don’t add initialize script

### ISSUE THREAD

**Issue by Kristinita** [oruelle/md_mermaid#8]
### 1. Summary

It would be nice, if would be possible to not add automatically a Mermaid initialize script to the HTML.

### 2. Argumentation

1. Adding `<script>` tag at the bottom of `<body>` is [**old**](https://stackoverflow.com/a/24070373/5951529) [**approach**](https://flaviocopes.com/javascript-async-defer/). Modern solution — adding it inside `<head>` tag.
1. Users can add scripts with their own methods. For example, I use [**defer.js**](https://github.com/shinsenter/defer.js/).

#### 3.1. Fully disable auto-adding of initialize script

In [**my fork**](https://github.com/Kristinita/md_mermaid/tree/KiraMermaidWorking) I [**remove these lines**](https://github.com/Kristinita/md_mermaid/blob/KiraMermaidWorking/md_mermaid.py) from `md_mermaid.py`:

```python
if is_mermaid:
    new_lines.append('')
    # This will initialize mermaid renderer. It's done only when the HTML document is ready,
    # to ensure the loading of mermaid.js file is finished.
    new_lines.append('''<script>
            function initializeMermaid() {
                mermaid.initialize({startOnLoad:true})
            }

            if (document.readyState === "complete" || document.readyState === "interactive") {
                setTimeout(initializeMermaid, 1);
            } else {
                document.addEventListener("DOMContentLoaded", initializeMermaid);
            }
    </script>''')
```

In my opinion, this is the best option. If you want, I make a pull request.

#### 3.2. Option

If you don’t want to delete the code above completely, it would be nice to have an option to not add a script. For example, if:

```yaml
md_mermaid:
  add_initialize_script: false
```

md_mermaid won’t add the script.

Thanks.

**Comment by tsbertalan** [comment]
Yeah, it would be nice to able to pass alternate options like [this](https://github.com/mermaid-js/mermaid/issues/580).
